### PR TITLE
created order page

### PR DESCRIPTION
**Pull request by VARSHAV2001**

### CODE REVIEW

File: client/src/pages/student/Order.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    level: "Its for beginners",
+    discription: "C is a general-purpose programming language.",
+    orders: "1",
+    price: "100",
+    img: "https://blog.ipleaders.in/wp-content/uploads/2021/05/online-course-blog-header-1024x576.jpg",
+  };
+
+  return (
+    <div>
+      <NavigationBar />
```
Comment (mishelshaji) [inline client/src/pages/student/Order.js]: Why don't you use a layout instead?

File: client/src/pages/student/Order.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      <div className="container">
+        <div className="row">
+          <div className="col-md-6 offset-md-3 mt-5">
+            <div class="shadow-lg p-3 rounded">
+              <p className="mb-3">Your Order</p>
+              <div className="row mb-3">
+                <div className="col-md-4">
+                  <img src={data.img} alt="" className="img-thumbnail" />
```
Comment (mishelshaji) [inline client/src/pages/student/Order.js]: Give a value for the alt attribute.

**Review by mishelshaji — CHANGES_REQUESTED** [review]

**Review by mishelshaji — APPROVED** [review]

File: client/src/pages/student/Order.js (hunk 2 above)
Comment (VARSHAV2001) [reply client/src/pages/student/Order.js]: given value for alt attribute

**Comment by mishelshaji** [comment]
Did you forget to commit changes?

**Review by mishelshaji — APPROVED** [review]

**Pull request merged**